src/Command.php

```diff
@@ -288,19 +288,22 @@ public function addArg($key, $value = null, $escape = null)
             setlocale(LC_CTYPE, $this->locale);
         }
         if ($value === null) {
-            // Only escape single arguments if explicitely requested
-            $this->_args[] = $escape ? escapeshellarg($key) : $key;
+            $this->_args[] = $doEscape ? escapeshellarg($key) : $key;
         } else {
-            $separator = substr($key, -1)==='=' ? '' : ' ';
             if (is_array($value)) {
-                $params = array();
                 foreach ($value as $v) {
-                    $params[] = $doEscape ? escapeshellarg($v) : $v;
+                    $this->addArg($key, $v, $escape);
```

Comment:
Here you've removed the feature that you can pass several `$value` like: `addArg('--optionX' , ['value1', 'value2'])` which gives `--optionX value1 value2`. So this would break BC. Actually there was a reason why it was implemented this way as some console commands have options that accept more than 1 value. I can't remember an example but can do some research. For now I'd say we should keep the old logic and fix it. What do you think?

Reply:
I agree, backward compatibility is important.

Reply:
Ok, apart from that the changes look good to me.

So as soon as you push the requested change I can merge and will create a new release. You can also add this to your DB now and I can add a link to that in the release notes if you want.

Reply:
Updated. Please take a look.